### ISSUE TITLE
Importing hashes saved locally via webui

### ISSUE THREAD

**Issue by relaxedricky** [hashview/hashview#322]
Hey,

Do we currently have a way to import hashes saved locally on the hashview box via the webui?

**Comment by i128** [comment]
Do you mean hashes that have already been cracked?

**Comment by relaxedricky** [comment]
No, recently we have been working through some of the old dumps and at times importing these through the webUI can cause things to fall over, the files can be up to and over 4GB at a time,

**Comment by i128** [comment]
Wow, okay. well short answer is no, there's no current way to do that. 
Long answer would be to manually import the hashes into the db under the hashes table and hashfilehashes table as well.
If you have it, a ruby stack error might help isolate where the bottle neck is. (presuming its either memory or timeout in webbrick)
